File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: list[int]) -> int:
        max1=0
        cur=0
        for i in nums:
            if i==1:
                cur+=1
            else:
                max1=max(max1,cur)
                cur=0
        return max(max1,cur)

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max=0,cur=0;
        for(int i:nums){
            if(i==1)
                cur+=1;
            else{
                max=Math.max(max,cur);
                cur=0;
            }
        }
        return Math.max(max,cur);
    }
}